index.html

```diff
@@ -29,7 +28,19 @@
                 <a href="#" class="logo-placeholder">Feven Seyfu</a>
             </div>
             <div class="hamburger-icon">
-                <img src="images/icon/menu-icon.png" class="menu-icon" alt="menu icon">
+                <a href="#"><img src="images/icons/Menu-icon.svg" class="menu-icon" alt="menu icon"></a>
+            </div>
+            <div class="desktop">
+                <ul class="nav-items">
+                    <li><a href="#" class="nav-item">Portfolio</a></li>
+                    <li><a href="#" class="nav-item">About</a></li>
+                    <li><a href="#" class="nav-item">Contact</a></li>
+                    <li class="nav-item">
+                        <a href="#">
+                            <img src="images/icons/Mail-icon.svg" alt="Mail icon" class="mail-icon">
+                        </a>
+                    </li>
+                </ul>
```

Comment:
You have two  `nav`  items, one for the desktop screen and the other for a small screen but both of the nav items are showing on top of each other on the desktop screen. See screenshot below

<img width="641" alt="image" src="https://user-images.githubusercontent.com/78151836/208214583-6ce7eb3d-a6e8-4488-a5dd-bb9216aa5577.png">

Check my comment below for a possible fix.

Reply:
Thank you for your feedback there were two nav items for the links that were duplicated while I was trying to fix merge conflicts. I have fixed the issue.